### PR TITLE
Release 3.1.1

### PR DESCRIPTION
**Pull request by lucacome**
Docs changes for `v1.1.1`

### CODE REVIEW

**Review by ADubhlaoich — APPROVED** [review]
LGTM!

File: docs/content/releases.md [hunk 1]
```diff
@@ -6,7 +6,39 @@ doctypes: ["concept"]
 toc: true
 docs: "DOCS-616"
 ---
+## NGINX Ingress Controller 3.1.1
+
+04 May 2023
+
+OVERVIEW:
+This release reverts the changes made in 3.1.0 to use sysctls to bind to lower level ports without the NET_BIND_SERVICE capability. It also adds support for serviceNameOverride in the Helm chart, that can be used to override the service name for the NGINX Ingress Controller. This is useful especially during an upgrade from versions prior to 3.1.0, to avoid downtime due to the service name change.
```
Comment (brianehlert) [inline docs/content/releases.md:14]: We should have a clear note here in the release note about how to "that can be used to override the service name for the NGINX Ingress Controller."  Inserted right after this overview.
Even if we outline the detail elsewhere.  Calling that behavior out in an obvious way is important to the issues that were reported.

File: docs/content/technical-specifications.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ docs: "DOCS-617"
 
 We advise users to run the most recent release of the NGINX Ingress Controller, and we issue software updates to the most recent release. We provide technical support for F5 customers who are using the most recent version of the NGINX Ingress Controller, and any version released within two years of the current release.
 
-Beginning with release 3.1.0 the `NET_BIND_SERVICE` capability is no longer used, and instead relies on `net.ipv4.ip_unprivileged_port_start` sysctl to allow port binding. Kubernetes 1.22 or later is required for this sysctl to be [classified as safe](https://kubernetes.io/docs/tasks/administer-cluster/sysctl-cluster/#safe-and-unsafe-sysctls).
+Beginning with release 3.1.1 the `NET_BIND_SERVICE` capability is no longer used, and instead relies on `net.ipv4.ip_unprivileged_port_start` sysctl to allow port binding. Kubernetes 1.22 or later is required for this sysctl to be [classified as safe](https://kubernetes.io/docs/tasks/administer-cluster/sysctl-cluster/#safe-and-unsafe-sysctls).
```
[the remark this answers is not in the document]
Comment (lucacome) [reply docs/content/technical-specifications.md:15]: good catch!

File: docs/content/releases.md (hunk 1 above)
Comment (lucacome) [reply docs/content/releases.md:14]: I've added the command

**Pull request merged**